waterlog/src/actions/MapActions.js

```diff
@@ -51,18 +51,62 @@ async function getData(dispatch){
     dispatch(fetchMapsDataFailure(error));
   });
 
-  return {segments, monitors, leaks}
+  return { segments, monitors, leaks }
 }
 
-export const fetchMapsData = () => (dispatch) => {
-	dispatch(fetchMapsDataBegin());
+function formatMapData(data) {
+  let markers, segments;
+  let todaysLeaks;
+  let date = new Date(Date.now());
+  if (!data.leaks) {
+    todaysLeaks = [];
+  } else {
+    todaysLeaks = data.leaks.filter(
+      leak =>
+        new Date(leak.latestTimeStamp).getDay() === date.getDay() &&
+        new Date(leak.latestTimeStamp).getMonth() === date.getMonth() &&
+        new Date(leak.latestTimeStamp).getFullYear() === date.getFullYear()
+    );
+  }
 
-  var data = getData(dispatch);
-  data.then(res =>{
-    dispatch(fetchMapsDataSuccess(res))
-  })
-  .catch(error => {
-    dispatch(fetchMapsDataFailure(error));
+  if (data.segments) {
+    segments = data.segments.map(seg => {
+      if (
+        JSON.stringify(Object.keys(seg)) !==
+        JSON.stringify(["id", "senseIDOut", "senseIDIn"])
+      ) {
+        return [];
+      }
+      const leak = todaysLeaks.find(leak => leak.segmentsId === seg.id);
+      if (leak) {
```

Comment:
Be consistent with your `if (bool) { }` bracket use

Reply:
I fixed that